2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -342,51 +584,55 @@ Nel terzo sprint, il gruppo svolge delle attività principalmente focalizzate su
-    [Banzato Alessio],  [3],  [/],    [/],    [/], [/],   [3],
-    [Carraro Riccardo], [/],  [/],    [4],    [/], [/],   [4],
-    [Gardin Giovanni],  [/],  [/],    [/],    [/], [3],   [3],
-    [Nardo Silvio],     [/],  [/],    [/],    [3], [/],   [3],
-    [Oseliero Antonio], [/],  [/],    [4],    [/], [/],   [4],
-    [Todesco Mattia],   [/],  [3],    [/],    [/], [/],   [3],
-    [Zaccone Rosario],  [/],  [3],    [/],    [/], [/],   [3],
-    [Totale ore ruolo], [3],  [6],    [8],    [3], [3],   [23],
-    [Costo orario],     [30,00], [20,00], [25,00], [25,00], [15,00], [],
-    [Costo ruolo], [90,00], [120,00], [200,00], [75,00], [45,00], [530,00]
+    columns: 8,
+    [*Membro*], [*Responsabile*], [*Amministratore*], [*Analista*], [*Progettista*], [*Programmatore*], [*Verificatore*], [*Totale*],
+    [Banzato],     [3],     [0],     [0],     [0],     [0],     [0],     [3],
+    [Carraro],     [0],     [0],     [4],     [0],     [0],     [0],     [4],
+    [Gardin],     [0],     [0],     [0],     [0],     [0],     [3],     [3],
+    [Nardo],     [0],     [0],     [0],     [3],     [0],     [0],     [3],
+    [Oseliero],     [0],     [0],     [4],     [0],     [0],     [0],     [4],
+    [Todesco],     [0],     [3],     [0],     [0],     [0],     [0],     [3],
+    [Zaccone],     [0],     [3],     [0],     [0],     [0],     [0],     [3],
+    [Totale ore],     [3],     [6],     [8],     [3],     [0],     [3],     [23],
+    [Costo ruolo],     [90],     [120],     [200],     [75],     [0],     [45],     [530],
   ),
-  caption: "Preventivo terzo sprint"
+  caption: "Prospetto del preventivo, sprint 3"
 )
-
-#set text(
-  size: 11pt
+#let data = (
+        ("Responsabile", 3),
+        ("Amministratore", 6),
+        ("Analista", 8),
+        ("Progettista", 3),
+        ("Programmatore", 0),
+        ("Verificatore", 3),
+)
+#figure({
+  roles-legend
+  canvas({
+    import draw: *
+    chart.piechart(..piechart-config, data)}
+  )},
+  caption: "Suddivisione oraria per ruolo, preventivo sprint 3",
+  kind: "chart",
+  supplement: "Grafico"
 )
 
-=== Diagramma di Gantt
-[da inserire]
-
-== Quarto Sprint dal 27-11-2023 al 04-12-2023
+== Sprint 4 dal 27-11-2023 al 04-12-2023
 
 === Obiettivi prefissati
 Gli obiettivi del quarto sprint si concentrano sull'aggiornamento e perfezionamento dei documenti redatti negli scorsi sprint, sulla finalizzazione di una prima versione
 del documento Analisi dei Requisiti e sull'esplorazione delle nuove tecnologie.
 
 Gli obiettivi dello sprint 4 sono:
-- effettuare una migliore pianificazione;
 - perfezionamento della stesura degli Use Case individuati finora;
 - redazione di una prima versione dell'Analisi dei Requisiti;
-- valutazione di un'eventuale data per un primo meeting con il professore Cardin per ottenere un feedback sull'Analisi dei Requisiti;
+- scelta di una data per un eventuale primo meeting con il professore Cardin per ottenere un feedback sull'Analisi dei Requisiti;
```

Comment:
per restare in linea con il resto del documento metterei "Professor Cardin" al posto di "professore Cardin" (anche se ho alcuni dubbi sull'utilizzo della maiuscola per Professor)